### PR TITLE
Add demo app for all desktops and instructions for bootstrapping

### PR DESCRIPTION
**Pull request by HankG**
This commit added platform projects for Windows and Linux for the built-in demo project. These platforms were tested under Linux Mint and Windows 10 running the latest Flutter SDK.

This commit also adds additional information to the README files in the pdf and demo project to point developers to the existence of the demo project. I honestly didn't realize there was such a detailed implementation in this repository. This would elevate this to the front of the pub.dev page as well as within this repository.

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+```
+
+...for Linux:
+```bash
+flutter run -d linux
+```
+
+...for macOS:
+```bash
+flutter run -d linux
```
Comment (DavBfr) [inline demo/README.md:57]: macos?

File: demo/android/app/src/main/kotlin/net/nfet/demo/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package net.nfet.demo
```
Comment (DavBfr) [inline demo/android/app/src/main/kotlin/net/nfet/demo/MainActivity.kt:1]: I'd prefere `printing_demo` as in the pubspec.yaml
Same everywhere else.

**Review by DavBfr — CHANGES_REQUESTED** [review]
A real dedicated application icon would be nice, instead of the default flutter.
I can also add the required authorization bits for iOS and macOS.

**Comment by HankG** [comment]
These latest updates to the demo app address:

- Removed the new Kotlin file which was autogenerated and unneeded
- Remove the default widget_test.dart file which was not being used and had compile time errors
- Add the print entitlement to the release macOS project. As far as I can tell that was the only missing config for enabling printing across the platforms. If there are more I can tweak the library documentation to make it more clear what is needed.

**Comment by DavBfr** [comment]
Merged, thanks.

**Pull request closed without merging**